### DISCUSSION TITLE
NextJS Port Not Forwarding in Codespaces Web

### DISCUSSION THREAD

**Discussion by sr5434** [community/community#123935] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

I am running ```npm run dev``` in my NextJS project in Codespaces. However, the port is not being forwarded and when I try to manually forward the port, Codespaces says "activating extensions" and does nothing.

**Comment by RysanekDavid** [comment, marked as answer]
Hey,

First check "package.json" 

**Should look like this:**

```
"scripts": {
  "dev": "next dev"
}
```

**Next make sure your Next.js configuration allows for connections from all interfaces**

```
module.exports = {
  devIndicators: {
    autoPrerender: false,
  },
  serverRuntimeConfig: {
    host: '0.0.0.0',
  },
};
```
**Also this modification in your package.json could help:**

```
"scripts": {
  "dev": "next dev -p 3000 -H 0.0.0.0"
}

```

Let me know if that works, David.
